pkg/apis/config/v1alpha1/adminqos.go

```diff
@@ -380,3 +436,5 @@ type NumaEvictionRankingMetric string
 // system level
 // +kubebuilder:validation:Enum=qos.pod;priority.pod;mem.usage.container
 type SystemEvictionRankingMetric string
+
+type ThresholdValue string
```

Comment:
it seems like this type is not used anywhere?

Reply:
ok